app/models/item.rb

```diff
@@ -2,15 +2,16 @@
 #
 # Table name: items
 #
-#  id              :bigint(8)        not null, primary key
+#  id              :integer          not null, primary key
```

Comment:
@seanmarcia Does this need to be addressed? I thought all `id` columns were bigint now? Did they get changed somewhere?

Reply:
Strange. I know any *new* rails app (5 or 5.1) will default to bigint. I wonder if this is something we need to alter in our schema?